tests/ctst/steps/pra.ts

```diff
@@ -328,8 +328,25 @@ Then('the kafka DR volume exists', { timeout: volumeTimeout + 2000 }, async func
     assert(volumeParsed.result!['volume phase'] === 'Bound');
 });
 
+Then('prometheus should scrap federated metrics from DR sink', { timeout: 70000 }, async function (this: Zenko) {
+    const prom = new PrometheusDriver({
+        endpoint: `http://${this.parameters.PrometheusService}:9090`,
+        baseURL: '/api/v1',
+    });
+
+    for (;;) {
```

Comment:
i don't think we should litter the code with "manual" timeout: if the logs are not correct, it should be improved in the framework instead

Reply:
I think the point was more: the test already manually set a 70s timeout, yet we are never calling `this.world.logger.debug` to log what we got when calling prometheus, so in case of timeout, we have 0 clue to understand the problem

Reply:
if we receive _anything_, we will bail out... so if we timeout, it means we did not receive anything : and the information to troubleshoot will be deep inside prometheus, not in the api call we make here, unfortunately...

Reply:
It might be a personal opinion, but I don't think an infinite loop is any better than a "manual" timeout

Reply:
> so not sure what could be logged...

If we can get some connectivity info then logging will be useful - if we expect nothing, the current approach is good enough...